app/views/mentors/check-your-answers.njk

```diff
@@ -49,6 +45,11 @@
           <a class="govuk-link" href="{{ actions.cancel }}">Cancel</a>
         </p>
       {% else %}
+
+        <p class="govuk-body">
+          Check that you typed in the teacher reference number (TRN) correctly.
+        </p>
```

Comment:
Should this also be added to the 'support' version of this page?

Reply:
Agreed. I've added the text to the support version.